codegen.yml

```
@@ -0,0 +1,23 @@
+overwrite: true
+schema: "http://localhost:8080/graphql"
+documents: './src/app/graphql/queries.ts'
```

```suggestion
documents: './src/app/graphql/*.graphql'
```

In coming time we will have multiple files for queries based on module name. So It would be better to use extension `.graphql` and  includes all the queries file based on extension.